#!/usr/bin/env python

from sulci.scripts.sulci_errors import siError

siError.main()

